src/EncompassRest/EncompassRestClient.cs

```diff
@@ -65,6 +65,8 @@ public static EncompassRestClient CreateFromAccessToken(string clientId, string
         private Webhook.Webhook _webhook;
         private Pipeline _pipeline;
         private BatchUpdate _batchUpdate;
+        private Contacts.BorrowerContacts _borrowerContacts;
```

Comment:
Can you add a `using EncompassRest.Contacts` directive at the top and remove the namespace from these fields and properties?

Reply:
Should webhook, schema, and loans be converted as well?

Reply:
those aren't possible because the class name is the same as the namespace.

Reply:
Remove namespace